src/alertsmanagement/azext_alertsmanagement/commands.py

```diff
@@ -12,17 +12,16 @@
 
 def load_command_table(self, _):
 
-    from ._client_factory import cf_action_rules
-    alertsmanagement_action_rules = CliCommandType(
-        operations_tmpl='azext_alertsmanagement.vendored_sdks.alertsmanagement.operations._action_rules_operations'
-                        '#ActionRulesOperations.{}',
-        client_factory=cf_action_rules)
-    with self.command_group('monitor action-rule', alertsmanagement_action_rules, client_factory=cf_action_rules,
-                            is_experimental=True) as g:
-        g.custom_command('create', 'create_alertsmanagement_action_rule')
-        g.generic_update_command('update', custom_func_name='update_alertsmanagement_action_rule',
-                                 setter_arg_name='action_rule', getter_name='get_by_name',
-                                 setter_name='create_update')
-        g.custom_command('delete', 'delete_alertsmanagement_action_rule')
-        g.custom_show_command('show', 'get_alertsmanagement_action_rule')
-        g.custom_command('list', 'list_alertsmanagement_action_rule')
+    from ._client_factory import cf_processing_rules
+    alertsmanagement_processing_rules = CliCommandType(
+        operations_tmpl='azext_alertsmanagement.vendored_sdks.alertsmanagement.operations._alert_processing_rules_operations'
+                        '#AlertProcessingRulesOperations.{}',
+        client_factory=cf_processing_rules)
+    with self.command_group('monitor alert-processing-rule', alertsmanagement_processing_rules, client_factory=cf_processing_rules,
+                            is_preview=True) as g:
+        g.custom_command('create', 'create_alertsmanagement_processing_rule')
+        g.generic_update_command('update', custom_func_name='update_alertsmanagement_processing_rule',
+                                setter_arg_name='alert_processing_rule', getter_name='get_by_name')
+        g.custom_command('delete', 'delete_alertsmanagement_processing_rule')
```

Comment:
Do we need to consider adding confirmation=True for the delete operation ?

Reply:
Thanks for the suggestion. Added confirmation to the delete command